### PR TITLE
Version 1.6.0

### PR DESCRIPTION
**Pull request by StephanSiemen**
For Python 3.9.0

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by StephanSiemen** [comment]
@conda-forge-admin, please rerender

**Comment by StephanSiemen** [comment]
Delete this because there is now an automatically generated PR https://github.com/conda-forge/ecmwf-api-client-feedstock/pull/9

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: d92bfa59a900921dd0f06bfe8b720ba857d59ada1a811241fbbea35fcfdc1836
+  sha256: 5236d23d1559cdfa4525701a86a1f3bb2b3994cdd0b9aa5762ddd6cef708b160
 
 build:
   number: 0
-  noarch: python
+  noarch: python >3.5
```
Comment (avalentino) [inline recipe/meta.yaml:14]: Specifying the minimum version here seems to cause problems.
IMHO having "python >3.5" in the requirements/host section is enough to build properly.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
     - pip
-    - python
+    - python >3.5
     - setuptools
   run:
     - python
```
Comment (avalentino) [inline recipe/meta.yaml:23]: Probably the constraint on the python version should be put also in this point

**Review by avalentino — CHANGES_REQUESTED** [review]

**Comment by avalentino** [comment]
Closing. V1.6.1 is already in now.

**Pull request closed without merging**